app/src/main/java/com/example/concordia_campus_guide/Activities/PathsViewModel.java

```diff
@@ -0,0 +1,29 @@
+package com.example.concordia_campus_guide.Activities;
+
+import android.content.Context;
+import android.graphics.Color;
+
+import androidx.lifecycle.ViewModel;
+
+import com.example.concordia_campus_guide.Global.SelectingToFromState;
+import com.example.concordia_campus_guide.Helper.PathFinder;
+import com.example.concordia_campus_guide.Models.Place;
+import com.example.concordia_campus_guide.Models.RoomModel;
+import com.example.concordia_campus_guide.Models.WalkingPoint;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class PathsViewModel extends ViewModel {
+
```

Comment:
If these are the only methods of this view model, could they be kept in the activity? We could create one less class like that. Thoughts?

Reply:
We could keep them to make it more extensible.